src/app/configuration/configuration.service.ts

```diff
@@ -40,6 +40,7 @@ export class ConfigurationService {
         'status': 'unread',
         'time': Date.now()
       };
+      this.configuration = configuration;
```

Comment:
This function should only create the notification on the parent planet.  We should not introduce the side effect of updating the configuration on the service as well.

In addition, this will only update the configuration when the configuration didn't exist on the parent and only for the open application.  Any other users who are logged in when the change will made will still see these errors.